scripts/e2e.sh

```diff
@@ -55,3 +57,32 @@ wait-for-pod-ready() {
     echo -n . 1>&2
   done
 }
+
+wait-for-container-terminated() {
+  local pod_name="${1}"
+  local container_name="${2}"
+  local timeout_secs=15
+  local increment_secs=1
+  local waited_time=0
+  local container_status
+
+  while [ ${waited_time} -lt ${timeout_secs} ]; do
+    container_status="$(get-pod-output-json "${pod_name}" | jq -r --arg container "${container_name}" ".status.containerStatuses[] | select(.name==\$container) | .state | keys[0]")"
+
+    if [ "${container_status}" == "terminated" ]; then
+      return 0
+    fi
+
+    sleep ${increment_secs}
+    (( waited_time += increment_secs ))
+
+    if [ ${waited_time} -ge ${timeout_secs} ]; then
+      echo
+      echo "'${container_name}' container never terminated. Last status was '${container_status}'."
```

Comment:
@sgoings good question; they do not.  Sample output from https://ci.deis.io/job/workflow-test-pr/4919/console

> 17:24:23 'workflow-dev-e2e' container never terminated. Last status was ''.

(output was produced when actual status logic still needed some work, but `echo` statement not changed.)